bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -349,6 +350,39 @@ func (s *ksServer) GetProjectLock(project string) *sync.Mutex {
 	return s.projectLocks[project]
 }
 
+// InstallIstio installs istio into the cluster.
+func (s *ksServer) InstallIstio(ctx context.Context, request CreateRequest) error {
+	regPath := s.knownRegistries["kubeflow"].RegUri
```

Comment:
How is the ISTIO manifest getting into bootstrap? Is it baked into the Docker image and we are using knownRegistries to locate the file?

Reply:
Yes, the same as deployment manager config